### PR TITLE
Emergency power unit

### PR DESCRIPTION
**Pull request by DebugOk**
## About the PR
Adds the Emergency Power Update, part of the "Fucking over salvage part 2" update.

TODO:
- [x] Get a proper sprite
- [x] Decide if ~15 minutes of power is enough for the high capacity version

## Why / Balance
I'm going to mess up their starting shuttle even more, but I don't want it running out of power too fast

Its literally just an SMES that cant be charged

## Media
Content Client_2ka3iLXIYG
Content Client_bGUc3NiXIb

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by deltanedas** [comment]
> Its literally just an SMES that cant be charged

mining station has this too for reference (unless you stole from there already :trollface:)

**Comment by DebugOk** [comment]
> > Its literally just an SMES that cant be charged
> 
> mining station has this too for reference (unless you stole from there already :trollface:)

Its pretty much the same idea :blunt:

**Comment by DebugOk** [comment]
Should be good to go

**Comment by DebugOk** [comment]
Probably gonna self merge tomorrow if its not reviewed before then:tm:

File: Resources/Textures/DeltaV/Structures/Power/epu.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/9c7d509354ee030300f63c701da63c17928c3b3b and modified by Swept | smes.png created by @ps3moira on Discord",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "smes"
+    },
+		{
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Power/epu.rsi/meta.json:13]: troll indent

Comment (DebugOk) [reply Resources/Textures/DeltaV/Structures/Power/epu.rsi/meta.json:13]: Real

**Comment by DebugOk** [comment]
Gotta figure out how to unfail the test first lol, I wonder if theres a better way to prevent deconstruction

**Comment by deltanedas** [comment]
mining station just has it not give you a board, so you can scrap it for steel and lv if you really dont need it

**Comment by DebugOk** [comment]
> mining station just has it not give you a board, so you can scrap it for steel and lv if you really dont need it

They completely copied the smes prototype, which I wanted to avoid if possible. But it seems like I'll have to if I want it to not give a board

**Comment by VMSolidus** [comment]
So I had originally been utterly confused why we seemingly want to screw over salvage players. And after playing cargo for a few rounds, I now understand that it's because they genuinely deserve it. This didn't have my approval last night, but it does now. Please do, you now have my support to do everything in your power to screw them over as much as possible.

**Comment by DebugOk** [comment]
> So I had originally been utterly confused why we seemingly want to screw over salvage players. And after playing cargo for a few rounds, I now understand that it's because they genuinely deserve it. This didn't have my approval last night, but it does now. Please do, you now have my support to do everything in your power to screw them over as much as possible.

Luckily I don't need approval and it would've been merged anyways, salvage will keep getting nerfs until they stop powergaming the shit out of everything they can get their hands on

**Comment by VMSolidus** [comment]
Good. They deserve it.

**Pull request closed without merging**